src/app/hostname-generators/views/HostnameGeneratorListView.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <RouteView
+    name="hostname-generator-list-view"
+    :params="{
+      name: '',
+      page: 1,
+      size: 15,
+    }"
+    v-slot="{ route, t, can, uri, me }"
+  >
+    <AppView
+      :docs="t('hostname-generators.href.docs')"
+    >
+      <template #title>
+        <h1>
+          <RouteTitle
+            :title="t('hostname-generators.routes.items.title')"
+          />
+        </h1>
+      </template>
+      <div class="stack">
+        <div v-html="t('hostname-generators.routes.items.intro', {}, { defaultMessage: '' })" />
+        <KCard>
+          <DataLoader
+            :src="uri(sources, '/hostnamegenerators', {}, {
```

Comment:
This is a super big nit, but! In our application we use the "machine-name" of `hostname-generator{s}` i.e. with a hyphen. We only use it without a hyphen here.

One of the purposes of having the `sources.ts` file with our own URIs is that we can totoally ignore whatever is made in the backend for whatever reason and define our own URIs so they work well for us. i.e. we have a thin proxy layer over the HTTP API.

Ideally we'd use `/hostname-generators` I would imagine? I've no idea why its `hostnamegenerators` in the backend but I would imagine the backend has a very good reason to make it like that (probably kube related). But I would also guess that reason doesn't apply to us, and seeing we use the hypenated version everywhere else in the frontend we might want to proxy `/hostname-generators` to `/hostnamegenerators` in the sources file. wdyt?

Reply:
That's a very good point 👍 I like to have the consistency and go for a unified use of the hyphenated version in the GUI 🙂